examples/pod_shell_crossterm.rs

```diff
@@ -1,7 +1,6 @@
 use futures::{channel::mpsc::Sender, SinkExt, StreamExt};
 use k8s_openapi::api::core::v1::Pod;
 
-#[cfg(unix)] use crossterm::event::Event;
```

Comment:
what's up with this? this will likely break windows

Reply:
when i build I got the warning
```
warning: unused import: `crossterm::event::Event`
 --> examples/pod_shell_crossterm.rs:4:18
  |
4 | #[cfg(unix)] use crossterm::event::Event;
  |                  ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
  ```
  
  maybe we should use `#[warn(unused_imports)]` instead if needed, I will update it likewise if the CI breaks 🤔

Reply:
hm, actually, this actually is unused. i thought it was some weirdness with an imported trait vs mac use or something. But this is not needed as it's just an unused enum, and terminal resize works without it. Probably an oversight in clippy not correctly running on forks.